ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+<!--
+Thank you for reporting an issue! For feature requests, scroll down.
+
+###########################
+Regarding problem reports:
+###########################
+
+Before you submit a problem report, please do the following:
+
+  * Search the list of existing issues to see if someone has reported the issue already.
+  * Update to the latest master branch of SeqAn and see if the issue persists.
+
+If you already know how to fix the issue, you can directly create a pull-request instead of an issue. See the guide on [CONTRIBUTING](./CONTRIBUTING.md) for more details.
+-->
+
+## Platform
+
+  - SeqAn version: <!-- enter version here -->
```

Comment:
```suggestion
  - [ ] SeqAn version: <!-- enter version here -->
```

Reply:
Why? This is the issue text that we are going to see, what's the point of us being able to check/un-check these? [Note that users creating the issue don't see checkboxes, they see the verbatim text inside the edit box]